ur_gazebo/launch/inc/ur_control.launch.xml

```diff
@@ -29,6 +29,9 @@
   <arg name="spawn_z" default="0.1" doc="At which height the model should be spawned. NOTE: lower values will cause the robot to collide with the ground plane." />
   <arg name="start_gazebo" default="true" doc="If true, Gazebo will be started. If false, Gazebo will be assumed to have been started elsewhere." />
 
+  <!--Setting initial configuration -->
+  <arg name="initial_joint_positions" default=" -J shoulder_pan_joint 0.0  -J shoulder_lift_joint -1.54 -J elbow_joint 1.54 -J wrist_1_joint -1.54 -J wrist_2_joint -1.54 -J wrist_3_joint 0.0" doc="Initial joint configuration of the robot"/>
```

Comment:
Could we maybe change that to an upright pose?
```suggestion
  <arg name="initial_joint_positions" default=" -J shoulder_pan_joint 0.0  -J shoulder_lift_joint -1.54 -J elbow_joint 0.0 -J wrist_1_joint -1.54 -J wrist_2_joint -1.54 -J wrist_3_joint 0.0" doc="Initial joint configuration of the robot"/>
```

Reply:
Yes sure